.github/workflows/cdk-sam-cli-integration-testing.yml

```diff
@@ -0,0 +1,54 @@
+name: Test the integration between SAM CLI and Latest CDK version
+
+on:
+  schedule:
+    - cron: "0 13 * * 1-5" # run 1PM UTC (6 am PDT) in every weekday
```

Comment:
Why run on a schedule instead of on pr?

Reply:
we already have the CDK integration test cases but we test against the synthesized templates. This integration was created to validate that every new CDK version is generating the template SAM CLI expects it, that is why we originally created the appveyor job to run daily instead of running it with each PR.